app/controllers/news_controller.rb

```diff
@@ -38,6 +38,8 @@ def show
 
   private
   def set_section
-    @section = SectionHome.find_by!(slug: params[:section]).section
+    Rollbar.silenced do
```

Comment:
We want the RecordNotFound exception to be handled by callers eg. Controllers so they can 404. But don't need rollbar spammed by 404 errors like /favicon.ico